row_035_run.py

```
@@ -0,0 +1,61 @@
+
+app = FastAPI()
+
+# Directory to save uploaded files
+UPLOAD_DIR = "uploads"
+os.makedirs(UPLOAD_DIR, exist_ok=True)
+
+# {fact rule=unrestricted-file-upload@v1.0 defects=1}
+EINK_MODEL = 'epd7in5b_V2'
+
+@app.post("/upload")
+async def upload_bmp(file: UploadFile = File(...)):
```

Ruff (0.13.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)